File: frontend/src/app/models/order.ts

export interface Order {

  id: number;
  tableId: number;   // <-- adiciona esse
  nTable: number;
  orderDate: string;
  quantities: number[];
  orderStatus: OrderStatus;
  productNames: string[];
  unitPrice: number[];
  totalAmount: number;
}

export enum OrderStatus {
  Preparing = 0,
  Ready = 1,
  Finalized = 2
}

File: backend/Model/Order.cs
using backend.DTO;

namespace backend.Model
{
    public class Order
    {
        public int Id { get; set; }

        public int TableId { get; set; }

        public DateTime OrderDate { get; set; }
        
        public decimal TotalAmount { get; set; }

        public virtual Table Table { get; set; }

        public virtual List<OrderItem> Items { get; set; } = new();
        
        public OrderStatus OrderStatus { get; set; } 
    }
    
    public enum OrderStatus
    {
        Processing,
        Complete,
        Finalized
    }
    

}
